react-native/react/tracker/render.desktop.js

```diff
@@ -12,8 +12,8 @@ import Proofs from './proofs-render'
 export default class Render extends BaseComponent {
   render () {
     return (
-      <div style={{backgroundColor: 'red', display: 'flex', flex: 1, flexDirection: 'column'}}>
-        <Header />
+      <div style={{display: 'flex', flex: 1, flexDirection: 'column'}}>
+        <Header {...this.props} />
```

Comment:
yeah, I came across a similar dilemma, I think this is okay since it's validated and documented by the proptypes.